Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -369,6 +369,38 @@ public override string ToString()
         }
     }
 
+    public class Ruler
+    {
+        public int length { get; set; }
+        
+        [IonField]
+        internal string unit;
+
+        [IonPropertyGetter("length")]
+        public int GetLength() 
+        {
+            return this.length + 10;
+        }
+
+        [IonPropertySetter("length")]
+        public void SetLength(int length) 
+        {
+            this.length = length - 10;
+        }
+        
+        [IonPropertyGetter("unit")]
+        public string GetUnit()
+        {
+            return this.unit == "cm" ? "centimeter" : this.unit;
```

Comment:
This is unnecessary complication for this test.

Reply:
This is done so we can verify that this method was actually used rather than the usual field serialization logic which would return "cm" (The check happens on line 158 in `IonObjectSerializerTest.cs`).